### PR TITLE
DBZ-3650 jenkins job for building rhel kafka image

### PR DESCRIPTION
**Pull request by novotnyJiri**
https://issues.redhat.com/browse/DBZ-3650

### CODE REVIEW

File: jenkins-jobs/job-dsl/downstream_rhel_prepare.groovy [hunk 1]
```diff
@@ -0,0 +1,32 @@
+pipelineJob('downstream-rhel-prepare-prepare-job') {
```
Comment (jcechace) [inline jenkins-jobs/job-dsl/downstream_rhel_prepare.groovy:1]: use ``rhel-downstram-prepare-job`` and name the fille ``rhel_downstream_prepare.groovy``

**Review by jcechace — CHANGES_REQUESTED** [review]

File: jenkins-jobs/job-dsl/downstream_rhel_prepare.groovy [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        stringParam('QUAY_ORGANISATION', '', "Organisation where images are copied")
+//        RHEL CONFIG
+        stringParam('RHEL_IMAGE', 'registry.access.redhat.com/ubi8:latest', "Base RHEL image")
+//        KAFKA CONFIG
+        stringParam('KAFKA_URL', '', "AMQ streams kafka")
+//        DEBEZIUM CONFIG
+        stringParam('DBZ_GIT_REPOSITORY', 'https://github.com/debezium/debezium.git', 'Repository from which Debezium sources are cloned')
+        stringParam('DBZ_GIT_BRANCH', 'master', 'A branch/tag of Debezium sources')
+//        DEBEZIUM CONNECT IMAGE CONFIG
+        textParam('DBZ_CONNECTOR_ARCHIVE_URLS', "", "List of URLs to productised Debezium connectors")
```
Comment (jpechane) [inline jenkins-jobs/job-dsl/downstream_rhel_prepare.groovy:21]: PLease be consistent in using single quotes to denote empty string

File: jenkins-jobs/job-dsl/downstream_rhel_prepare.groovy [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+//        EXTRA CONFIG
+        textParam('DBZ_EXTRA_LIBS', "", "List of extra libraries added to connectors")
```
Comment (jpechane) [inline jenkins-jobs/job-dsl/downstream_rhel_prepare.groovy:23]: See above

**Review by jpechane — APPROVED** [review]
@novotnyJiri LGTM, I've left just minor comments related to string quotes usage. When corrected and Jakub's comments are incorporated then we are good to go.

**Comment by jpechane** [comment]
@novotnyJiri Applied, thanks!

**Pull request merged**